### PR TITLE
Allowing editors to move stories between areas

### PR DESCRIPTION
**Pull request by fredriklindman**
Editors now have an option to move stories between areas through an alternative in the new options menu. 
If a dynamic attribute is missing in the target area, it will get copied to the new area.

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/header.jsp [hunk 1]
```diff
@@ -31,17 +32,35 @@ THE SOFTWARE.
...
 <div id="settings-div" class="dropdown dropdown-tip">
-        <ul class="dropdown-menu">
-        <li>
-            <a id="login-out" class="" href="../auth/logout">
-                <c:if test="${isLoggedIn == true}">LOG OUT</c:if>
-                <c:if test="${isLoggedIn == false}">LOG IN</c:if>
-            </a>
+    <ul class="dropdown-menu">
+        <li><a id="login-out" class="" href="../auth/logout"> <c:if
+                    test="${isLoggedIn == true}">LOG OUT</c:if> <c:if
```
Comment (nilsi) [inline src/main/webapp/WEB-INF/views/header.jsp]: Indentions here looks a bit strange

**Comment by nilsi** [comment]
Some general comments. Mostly small stuf, overall it work's great!

I dont really like the popup dialog box. I dont see the use of it. it would look more smooth with a send button in connection to the drop down list. Something like bootstraps dropdown in a button group. Where you select area and then push the send button.

Sometimes it "thinks" that I have one selected item even though I don't have it. For example if you send one item and then trying to send again. And sometimes I get a popup saying that I dont have any items selected.

I also experienced that not all items that are moved are placed at the bottom. Sometimes they are getting mixed. I don't know how or why this happens and it's kind of hard to reproduce. But I manage to do it a couple of times when moving multiple stories to another area

**Comment by fredriklindman** [comment]
Thank you for finding the bug with mixing the prio, I think I know what's wrong and will fix it tomorrow. 

About the popup, I discussed it a lot with Gustaf and he strongly advised me to have it like this. I had a button like you say but then we started to check out how similar things work in Gmail and came up with this. The popup should not interrupt daily work flow since this is a very unusual button. 

The stories are in fact selected even though they have been moved but I agree it could be confusing if users try to move them again. However I thought it was nice to have the moved stories selected when opening the new area after a move. I can simply empty the list of selected stories after a move if you prefer that? Or we could make the selected stories area specific but that would require some more work, to do later maybe.

**Comment by nilsi** [comment]
1. No problem :-)
2. Okey two against one then. You can keep it there. I just don't see the use of it because you cant really hurt anything by moving items, I see the use of it when deleting items where you really could get some annoying extra work if you accidentally delete a story.
3. For me it feel most natural to empty the list after moving it.

**Comment by fredriklindman** [comment]
1. Okay, I can understand both sides but I leave it like this then. What has to be mentioned though is that you can actually mess up some things since attribute options are cloned to the new area and thenes/epics are cloned as well. Not as much mess as deleting but still some mess ;-)
2. Alright, then I will empty it.

--- Originalmeddelande ---

Från: "Nicklas Nilsson" notifications@github.com
Skickat: 16 juni 2013 19:34
Till: "sonyxperiadev/BacklogTool" BacklogTool@noreply.github.com
Kopia: "Fredrik Persson" fredrik-persson@outlook.com
Ämne: Re: [BacklogTool] Allowing editors to move stories between areas (#68)
1. No problem :-)
2. Okey two against one then. You can keep it there. I just don't see the use of it because you cant really hurt anything by moving items, I see the use of it when deleting items where you really could get some annoying extra work if you accidentally delete a story.
3. For me it feel most natural to empty the list after moving it.

---

**Comment by fredriklindman** [comment]
I have updated after review and it's ready for a new try ;-)

**Comment by nilsi** [comment]
Just one small thing then it's ready to merge. Since we are using JQuery UIs dialog boxes on all places, I think we should use that also on the "no selected stories" alert box.

**Pull request merged**